File: nodejs/dse_delete.js
const init_connection = require('./ip_address')

connection = init_connection()

// this is a delete statement in nodejs
const query1 = 'DELETE FROM crud.users WHERE name = BOB';
connection.client.execute(query1)
.then(result => console.log(result))
.catch((error) => {console.log(error)});

File: python/dse_delete.py
#!/usr/bin/env python3

# # this is a delete statement in python
from dse.cluster import Cluster, ExecutionProfile, EXEC_PROFILE_DEFAULT
from dse.query import tuple_factory
from ip_address import connection

cluster = Cluster([connection.ip_address])
# session = cluster.connect()

# print session.execute("DELETE FROM crud.users WHERE name = BOB")[0]